src/jsSSE.php

```diff
@@ -107,6 +128,15 @@ private function output(string $content): void
      */
     public function sendBlock(string $id, string $data, string $name = null): void
     {
+        if (connection_aborted()) {
+            $this->hook('aborted');
+
+            // stop execution when aborted if not keepAlive.
+            if (!$this->keepAlive) {
+                exit();
```

Comment:
i think this must be `$this->app->terminate()`

Reply:
Good point @abbadon1334 , and as we are in js context, maybe even `terminateJson()` may be needed.

Reply:
I made the change using app->callExit(), so beforeExit trigger. There is no need to use app->terminate because, at this point, the user has disconnect the browser, so there is nothing to send to it.

Reply:
@ibelar LGTM

@mvorisek terminateJson i think is not needed, headers are already set and we need only to exit as usual with a `$content = ''` which is done in terminate